src/com/facebook/buck/apple/project_generator/ProjectGenerator.java

```diff
@@ -2492,6 +2532,11 @@ private Path getObjcModulemapVFSOverlayLocationFromSymlinkTreeRoot(Path headerSy
     return headerSymlinkTreeRoot.resolve("objc-module-overlay.yaml");
   }
 
+  private Path getTestingModulemapVFSOverlayLocationFromSymlinkTreeRoot(
+      Path headerSymlinkTreeRoot) {
+    return headerSymlinkTreeRoot.resolve("testing.yaml");
```

Comment:
I'd be consistent with the naming of `objc-module-overlay.yaml` and name this `testing-overlay.yaml`.

Reply:
👍